.travis.yml

```diff
@@ -8,6 +8,10 @@ before_install:
 - pip install -U pytest pytest-cov
 - pip install codecov
 - sudo apt-get install lilypond
+- sudo apt-get install musescore
+- sudo apt-get install xvfb
+- Xvfb :0 -screen 0 1280x768x24&
```

Comment:
Mind clarifying in comment what 1280x768x24& is referring to?

Reply:
it does not quite matter; xvfb is used to simulate a window opened for musescore.

Reply:
1280x768x24& is the param for that fake window